cypress/integration/TextArea.spec.ts

```diff
@@ -53,4 +53,18 @@ describe('Text Area', () => {
       getTextArea().should('be.disabled');
     });
   });
+
+  context(`given the 'Placeholder' story is rendered`, () => {
+    beforeEach(() => {
+      h.stories.load('Components|Inputs/TextArea/React/Top Label', 'With Placeholder');
+    });
+
+    it('should pass accessibility checks', () => {
+      cy.checkA11y();
+    });
+
+    it('should render a placeholder text', () => {
+      getTextArea().should('have.attr', 'placeholder', 'Placeholder');
+    });
+  });
```

Comment:
Curious if we should add another test here to assert when you type into the input or text area, it should display that text? thoughts @NicholasBoll ?

Reply:
Yes, the tests against the other stories include making sure the input gets focus as well as typing in it and making sure the value changes